src/client/refresh/soft/sw_main.c

```diff
@@ -1469,6 +1470,10 @@ static int RE_PrepareForWindow(void)
 	return flags;
 }
 
+// struct for save refexport callbacks, copy of re struct from main file
+// used different variable name for prevent confusion
+static refexport_t	refexport;
```

Comment:
What's the point of this change?
refexport is only used in GetRefAPI() and returned by value, I don't understand why it should be a global/static variable

Reply:
@DanielGibson Oops, thank you, i will fix.